### ISSUE TITLE
Manuscript

### ISSUE THREAD

**Issue by mhesselbarth** [equitable-equations/fqar#14]
https://github.com/openjournals/joss-reviews/issues/6366

Line 8: Given that non-native species are inlcuded by scored 0, I would reword the sentence and remove the first "native"

Line 41: How would a misclassified species be identified? Only by using certain outliers assuming they are wrong? However, they could also be "true" outliers, correct? 

Misc: How does the package relate to other, existing packages, such as https://cran.r-project.org/web/packages/fqacalc/index.html?

**Comment by equitable-equations** [comment]
@mhesselbarth thanks for your work! I'll revise to reflect these and other points over the weekend.

**Comment by equitable-equations** [comment]
@mhesselbarth Thanks again for your patience as I reconciled this package with new behaviors from the API with which it interacts. I've update the paper to reflect your suggestions and will push these to the `revision` branch shortly. 

Line 8: Given that non-native species are inlcuded by scored 0, I would reword the sentence and remove the first "native." **Done**

Line 41: How would a misclassified species be identified? Only by using certain outliers assuming they are wrong? However, they could also be "true" outliers, correct? **A sentence has been added to clarify this point. We are currently revising on a paper addressing this question in much greater depth.**

Misc: How does the package relate to other, existing packages, such as https://cran.r-project.org/web/packages/fqacalc/index.html? **Added the following sentence to the `need` section: This package compliments existing R packages for floristic quality analysis, including `fqacalc` and `fqadata`, which support the work of field practitioners wishing to make use of R. **
